src/main/java/racing/domain/Car.java

```diff
@@ -6,15 +6,33 @@
 
     private final CarIndex index;
     private final MoveStrategyAble moveStrategy;
+    private final String name;
 
-    public Car(MoveStrategyAble moveStrategy) {
-        this.index = new CarIndex();
+    public Car(int index, MoveStrategyAble moveStrategy, String name) {
+        this.index = new CarIndex(index);
         this.moveStrategy = moveStrategy;
+        this.name = name;
+
+        validateCarName();
     }
 
-    public Car(int index, MoveStrategyAble moveStrategy) {
-        this.index = new CarIndex(index);
+    public Car(MoveStrategyAble moveStrategy, String name) {
+        this.index = new CarIndex();
         this.moveStrategy = moveStrategy;
+        this.name = name;
+
+        validateCarName();
+    }
```

Comment:
11~16라인과 코드 중복이 발생하고 있네요
```suggestion
    public Car(MoveStrategyAble moveStrategy, String name) {
        this(new CarIndex(), moveStrategy, name);
    }
```